sup3r/preprocessing/data_handling.py

```diff
@@ -437,12 +437,12 @@ class DataHandler(FeatureHandler, InputMixIn):
     def __init__(self, file_paths, features, target=None, shape=None,
                  max_delta=20, temporal_slice=slice(None),
                  hr_spatial_coarsen=None, time_roll=0, val_split=0.05,
-                 sample_shape=(10, 10, 1), raster_file=None,
+                 sample_shape=(10, 10, 1), raster_file=None, raster_index=None,
                  shuffle_time=False, time_chunk_size=None, cache_pattern=None,
-                 overwrite_cache=False, load_cached=False,
-                 train_only_features=None, max_workers=None,
+                 overwrite_cache=False, overwrite_ti_cache=False,
+                 load_cached=False, train_only_features=None, max_workers=None,
                  extract_workers=None, compute_workers=None, load_workers=None,
-                 norm_workers=None, ti_workers=None):
+                 norm_workers=None, ti_workers=None, handle_features=None):
```

Comment:
docstring for handle_features? why provide separately?

Reply:
This was to enable precomputing for the forward pass pipeline. When I was getting threads working these repeated calls to xarray caused issues.